cli/man/splinter-keygen.1.md

```diff
@@ -0,0 +1,78 @@
+% SPLINTER-KEYGEN(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**splinter-keygen** — Generates user and daemon keys for Splinter
+
+SYNOPSIS
+========
+
+**splinter keygen** \[**FLAGS**\] \[**OPTIONS**\] \[*KEY-NAME*\]
+
+DESCRIPTION
+===========
+
+This command generates secp256k1 public/private keys for Splinter.
+
+The `--system` flag generates keys for the Splinter daemon (`splinterd`) that
+are stored in `/etc/splinter/keys`.
+
+If no option is specified, this command generates user keys that are stored in
+the directory `$HOME/splinter/keys`.
+
+FLAGS
+=====
+
+`-h`, `--help`
+: Prints help information
+
+`-q`, `--quiet`
+: Decreases verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`--system`
+: Generates system keys for `splinterd` in `/etc/splinter/keys`.
+  By default, the key files are named `root.priv` and `root.pub`, because
+  this command runs as root when the `--system` option is used.
+
+`-V`, `--version`
+: Prints version information
+
+`-v`
+: Increases verbosity (the opposite of -q). Specify multiple times for more
+  output.
+
+ARGUMENTS
+=========
+
+`*KEY-NAME*`
+: (Optional) Specifies the base name for the key files. By default, the user
+  name is used.
+
+EXAMPLES
+========
+
+This example generates user keys for a Splinter user who is logged in as
+`paulbunyan`.
+
+```
+$ splinter keygen
+writing file: "/Users/paulbunyan/splinter/keys/paulbunyan.priv"
+writing file: "/Users/paulbunyan/splinter/keys/paulbunyan.pub"
+```
+
+The next example generates system keys for the Splinter daemon, but specifies
+`splinterd` as the basename for the files (instead of the default `root`).
```

Comment:
This is misleading as the default is the username, not 'root'.

Reply:
Changed to "(instead of the user name").